sdk/url_values.go

```diff
@@ -14,6 +14,45 @@ func addMapToUrlValues(data map[string]any, values *url.Values) {
 		}
 
 		switch v := value.(type) {
+		case []string:
+			for _, s := range v {
+				values.Add(key, s)
+			}
+			continue
+		case *[]string:
+			if v == nil {
+				continue
+			}
+			for _, s := range *v {
+				values.Add(key, s)
+			}
+			continue
+		case []any:
+			tmp := make([]string, 0, len(v))
+			ok := true
+			for _, e := range v {
+				switch s := e.(type) {
+				case string:
+					tmp = append(tmp, s)
+				case *string:
+					if s != nil {
+						tmp = append(tmp, *s)
+					} else {
+						ok = false
+					}
+				default:
+					ok = false
+				}
+				if !ok {
+					break
+				}
+			}
+			if ok {
+				for _, s := range tmp {
+					values.Add(key, s)
+				}
+			}
```

Comment:
Instead of having the `ok` variable, we could just do `if len(tmp) > 0`

Reply:
Please see commit https://github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/pull/164/commits/e59106abdd4c9bd05bc6a9eef30268887fe14e7f